packages/tizen_notification/lib/notification.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:flutter/services.dart';
+import 'package:tizen_app_control/app_control.dart';
+
+import 'src/types.dart';
+
+export 'package:tizen_app_control/app_control.dart';
+
+export 'src/enums.dart';
+export 'src/types.dart';
+
+// ignore: public_member_api_docs
+class TizenNotificationDetails {
+  /// [TizenNotificationDetails] contains detail information about the notification
+  /// being sent.
+  ///
+  /// [images] only supports type [List<NotificationImage>] and [NotificationImage].
+  const TizenNotificationDetails({
+    this.images,
+    this.properties,
+    this.appControl,
+    this.vibration,
+    this.sound,
+    this.displayApplist = DisplayApplist.all,
+    this.onGoing = false,
+  });
+
+  /// A single or a list of [NotificationImage].
+  ///
+  /// [NotificationImage] refers to any image shown on the notification panel.
+  final dynamic images;
+
+  /// Properties that configure how the system handles notification at
+  /// certain scenarios.
+  ///
+  /// [properties] can have multiple [Property] values with the bitwise OR operator.
+  /// For example, [Property.disableAppLaunch] | [Property.volatileDisplay].
+  final int? properties;
+
+  /// A set of information used by app control to launch other applications.
+  final AppControl? appControl;
+
+  /// The vibration triggered when a notification is shown on the panel.
+  final NotificationVibration? vibration;
```

Comment:
What's the default behavior if this argument is null? No vibration?

Reply:
Yes, no vibration.